src/components/forms/PlanForm/helpers.ts

```diff
@@ -245,10 +245,7 @@ export const DynamicMDAActivities = pick(planActivities, [
   DYNAMIC_MDA_COMMUNITY_DISPENSE_ACTIVITY_CODE,
   DYNAMIC_FAMILY_REGISTRATION_ACTIVITY_CODE,
 ]);
-export const DynamicIRSActivities = pick(planActivities, [
-  DYNAMIC_BCC_ACTIVITY_CODE,
-  DYNAMIC_IRS_ACTIVITY_CODE,
-]);
```

Comment:
@moshthepitt also removing `DYNAMIC_BCC_ACTIVITY_CODE` from  here is causing changes on the snapshots.